src/AzStorage.jl

```diff
@@ -284,6 +284,62 @@ x = read!(io, zeros(10))
 """
 Base.write(o::AzObject, data) = write(o.container, o.name, data)
 
+"""
+    writedlm(container, "blobname")
+
+Write the array `data` to a delimited blob with the name `blobname` in container `container::AzContainer`
+"""
+function DelimitedFiles.writedlm(c::AzContainer, o::AbstractString, data::AbstractArray)
+    tfile = tempname()
+    DelimitedFiles.writedlm(tfile,data)
+    write(c,o,read(tfile))
+    rm(tfile)
+end
+
+"""
+    writedlm(io:AzObject, data)
+
+write the array `data` to `io::AzObject`
+
+# Example
+```
+io = open(AzContainer("mycontainer";storageaccount="mystorageaccount"), "foo.txt")
+writedlm(io, rand(10,10))
+x = readdlm(io)
+```
+"""
+function DelimitedFiles.writedlm(o::AzObject, data::AbstractArray)
+     writedlm(o.container, o.name, data)
+end
+
+"""
+    readdlm(container, "blobname")
+
+Read the data in a delimited blob with the name `blobname` in container `container::AzContainer`
+"""
+function DelimitedFiles.readdlm(c::AzContainer, o::AbstractString)
+    tfile = tempname()
```

Comment:
same comment as for writedlm where we can use an IOBuffer.  So something like:
```julia
io=IOBuffer(;write=true,read=true)
write(io,read(c, o, String))
seekstart(io)
readdlm(io)

Reply:
changed to IOBuffer and confirmed readdlm/writedlm are working correctly